apps/nextjs/src/components/Onboarding/AcceptTermsForm.tsx

```diff
@@ -0,0 +1,138 @@
+"use client";
+
+import { useState } from "react";
+
+import { useUser } from "@clerk/nextjs";
+import logger from "@oakai/logger/browser";
+import {
+  OakBox,
+  OakFlex,
+  OakHeading,
+  OakLink,
+  OakP,
+  OakPrimaryButton,
+  OakSpan,
+} from "@oaknational/oak-components";
+import { useReloadSession } from "hooks/useReloadSession";
+import Link from "next/link";
+
+import Button from "@/components/Button";
+import CheckBox from "@/components/CheckBox";
+import SignUpSignInLayout from "@/components/SignUpSignInLayout";
+import TermsContent from "@/components/TermsContent";
+import { trpc } from "@/utils/trpc";
+
+export const AcceptTermsForm = () => {
+  const [dropDownOpen, setDropDownOpen] = useState(true);
+  const { isLoaded } = useUser();
+  const reloadSession = useReloadSession();
+
+  const [termsAcceptedLocal, setTermsAcceptedLocal] = useState(false);
+  const [privacyAcceptedLocal, setPrivacyAcceptedLocal] = useState(false);
+  const setDemoStatus = trpc.auth.setDemoStatus.useMutation();
+  const acceptTerms = trpc.auth.acceptTerms.useMutation({});
+
+  const handleAcceptTermsOfUse = async () => {
+    try {
+      await setDemoStatus.mutateAsync();
+      logger.debug("Demo status set successfully");
+
+      const response = await acceptTerms.mutateAsync({
+        termsOfUse: new Date(),
+        privacyPolicy: privacyAcceptedLocal ? new Date() : false,
+      });
+
+      if (!response?.acceptedTermsOfUse) {
+        throw new Error("Could not accept terms of use");
+      }
+      logger.debug("Terms of use accepted successfully.");
+
+      await reloadSession();
+      logger.debug("Session token refreshed successfully. Redirecting");
+
+      window.location.href = "/";
+    } catch (error) {
+      logger.error(error, "An error occurred while accepting terms of use");
+    }
+  };
+
+  return (
+    <SignUpSignInLayout loaded={isLoaded}>
+      <OakBox
+        $mh="auto"
+        $borderRadius="border-radius-m"
+        $background="white"
+        $pa="inner-padding-xl2"
+        $maxWidth={"all-spacing-22"}
+      >
+        <OakHeading $font="heading-6" tag="h1">
+          This product is experimental and uses AI
+        </OakHeading>
+        <OakBox $mt="space-between-s">
+          <OakP>
+            We have worked to ensure that our tools are as high quality and as
+            safe as possible but we cannot guarantee accuracy. Please use with
+            caution.
+          </OakP>
+        </OakBox>
+
+        <OakBox $pt="inner-padding-m" $mv="space-between-s">
+          <CheckBox
+            label="Accept"
+            setValue={setPrivacyAcceptedLocal}
+            size="base"
+          >
+            <OakSpan>
+              Keep me updated with latest Oak AI experiments, resources and
+              other helpful content by email. You can unsubscribe at any time.
+              See our{" "}
```

Comment:
The language here shifts from "me" - the user to "our" - Oak. Should this be "the"?

Reply:
This isn't new copy, but you are right that the grammar isn't quite right

The previous sentence has already switched to "You", so I don't think it's as bad as it looks at first glance

That said, "Keep me updated **with latest** Oak AI experiments" seems to be missing a "the"?